conf/layer.conf

```diff
@@ -19,7 +19,7 @@ BBFILE_PRIORITY_meta-hdc = "7"
 # cause compatibility issues with other layers
 LAYERVERSION_meta-hdc = "1"
 
-LAYERDEPENDS_meta-hdc = "core openembedded-layer"
+LAYERDEPENDS_meta-hdc = "core openembedded-layer meta-python networking-layer"
 
```

Comment:
This will break the IDP build which uses the old WRL7 build work flow.  You can bitbake add layer for this.